apps/rush/src/actions/GenerateAction.ts

```diff
@@ -59,13 +58,13 @@ export default class GenerateAction extends CommandLineAction {
     const stopwatch: Stopwatch = Stopwatch.start();
     const isLazy: boolean = this._lazyParameter.value;
 
-    console.log('Starting "rush generate"' + os.EOL);
-
-    if (this._rushConfiguration.packageReviewFile) {
-      this._packageReviewChecker = new PackageReviewChecker(this._rushConfiguration);
-      this._packageReviewChecker.saveCurrentDependencies();
+    if (this._rushConfiguration.approvedPackagesPolicyEnabled) {
+      const packageReviewChecker: PackageReviewChecker = new PackageReviewChecker(this._rushConfiguration);
+      packageReviewChecker.saveCurrentDependencies();
     }
```

Comment:
For now, I just reduced PackageReviewChecker to a static class.

---
In reply to: [116136972](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116136972) [](ancestors = 116136972,116121866)